src/langgraph-platform/use-remote-graph.mdx

```diff
@@ -1,289 +1,300 @@
+from langgraph.pregel.remote import RemoteGraph
 
-<Tabs>
-    <Tab title="Python">
-    ```python
-    from langgraph_sdk import get_client, get_sync_client
-    from langgraph.pregel.remote import RemoteGraph
+url = <DEPLOYMENT_URL>
+client = get_client(url=url)
+sync_client = get_sync_client(url=url)
 
-    url = <DEPLOYMENT_URL>
-    client = get_client(url=url)
-    sync_client = get_sync_client(url=url)
+# Using graph name (uses default assistant)
+graph_name = "agent"
+remote_graph = RemoteGraph(graph_name, client=client, sync_client=sync_client)
 
-    # Using graph name (uses default assistant)
-    graph_name = "agent"
-    remote-graph = RemoteGraph(graph_name, client=client, sync_client=sync_client)
+# Using assistant ID
+assistant_id = <ASSISTANT_ID>
+remote_graph = RemoteGraph(assistant_id, client=client, sync_client=sync_client)
+```
 
-    # Using assistant ID
-    assistant_id = <ASSISTANT_ID>
-    remote-graph = RemoteGraph(assistant_id, client=client, sync_client=sync_client)
-    ```
-    </Tab>
-    <Tab title="JavaScript">
-    ```ts
-    import { Client } from "@langchain/langgraph-sdk";
-    import { RemoteGraph } from "@langchain/langgraph/remote";
+```typescript JavaScript
+import { Client } from "@langchain/langgraph-sdk";
+import { RemoteGraph } from "@langchain/langgraph/remote";
 
-    const client = new Client({ apiUrl: `<DEPLOYMENT_URL>` });
+const client = new Client({ apiUrl: `<DEPLOYMENT_URL>` });
 
-    // Using graph name (uses default assistant)
-    const graphName = "agent";
-    const remoteGraph = new RemoteGraph({ graphId: graphName, client });
+// Using graph name (uses default assistant)
+const graphName = "agent";
+const remoteGraph = new RemoteGraph({ graphId: graphName, client });
 
-    // Using assistant ID
-    const assistantId = `<ASSISTANT_ID>`;
-    const remoteGraph = new RemoteGraph({ graphId: assistantId, client });
-    ```
-    </Tab>
-</Tabs>
+// Using assistant ID
+const assistantId = `<ASSISTANT_ID>`;
+const remoteGraph = new RemoteGraph({ graphId: assistantId, client });
+```
 
-## Invoking the graph
+</CodeGroup>
 
-Since `RemoteGraph` is a `Runnable` that implements the same methods as `CompiledGraph`, you can interact with it the same way you normally would with a compiled graph, i.e. by calling `.invoke()`, `.stream()`, `.get_state()`, `.update_state()`, etc (as well as their async counterparts).
+## Invoke the graph
+
+`RemoteGraph` implements the same Runnable interface as `CompiledGraph`, so you can use it in the same way as a compiled graph. It supports the full set of standard methods, including `.invoke()`, `.stream()`, `.get_state()`, and `.update_state()`, as well as their asynchronous variants.
 
 ### Asynchronously
 
 <Note>
 To use the graph asynchronously, you must provide either the `url` or `client` when initializing the `RemoteGraph`.
+To use the graph asynchronously, you must provide either the `url` or `client` when initializing the `RemoteGraph`.
```

Comment:
Double text. We can remove this line.

<img width="1546" height="338" alt="image" src="https://github.com/user-attachments/assets/dee7c42b-ef9d-4276-95f5-405986a44948" />